.editorconfig

```diff
@@ -0,0 +1,8 @@
+root = true
+
+[*.html]
+indent_style = space
+indent_size = 4
+charset = utf-8
+trim_trailing_whitespace = false
+insert_final_newline = false
```

Comment:
This file will enforce the formatting of the `.html` file so VScode will not prefer one formatting over another in case they are set differently per user. For example, my editor is set up for `indent_size = 2` so without this, If I hit the format button, the code will be formatted completely different than on someone else's computer and result in a lot of extra lines of change just because the spacing is different even though the content didn't change